app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
```

Comment:
```suggestion
    for i in range(len(argv) - 1):
```
`_` is used in for loop, when the variable is not needed in the loop body. In your case it's just the opposite